wiki/en/Running-a-Server.md

```diff
@@ -100,12 +89,14 @@ If you want to run a Server on a Raspberry Pi, you will need to [compile from so
 1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
 1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
 
-### Configuration
-
 You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
 
 `sudo systemctl stop jamulus-headless`
 
+_To upgrade your Server, just repeat the steps above._
+
+### Configuration
+
 #### Running in Registered mode
 
 The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
```

Comment:
> to [run a Registered Server](Running-a-Server#server-types):

Only the server type should be linked: "to run a [Registered Server](Running-a-Server#server-types):"

Reply:
I'm still not happy with lines 103-134.  As a _guide_, this should go into more depth about what the options mean, rather than give what looks like a "copy-n-paste this and run your server now!" command.

"Configuration" is where we should be explaining each option that can be set that affects how a server is run and how that option can be set, either from the command line or, when available, from the UI.  I really don't like the split between the two as things stand.

Reply:
The link explains _what_ a Registered Server _is_ -- not _how to run_ a Registered Server.  That's why it shouldn't include "run a" in the link.